src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FtpaDecisionMidEventTest.java

```diff
@@ -169,4 +167,447 @@ public void should_successfully_record_ftpa_respondent_decision() {
         final Set<String> errors = callbackResponse.getErrors();
         assertThat(errors).isEmpty();
     }
+
+    @Test
+    public void should_successfully_set_ftpa_appellant_decision_reasons_notes_visibility_when_granted() {
+
+        when(callback.getEvent()).thenReturn(Event.LEADERSHIP_JUDGE_FTPA_DECISION);
+        when(asylumCase.read(FTPA_APPLICANT_TYPE, String.class)).thenReturn(Optional.of("appellant"));
```

Comment:
"appellant" - Shouldn't this be a constant ?

Reply:
same as above